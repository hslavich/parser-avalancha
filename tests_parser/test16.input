
-- Funciones (reglas)

fun yy
  True  , x -> x
  False  -> False

