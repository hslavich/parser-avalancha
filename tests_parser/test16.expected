NameError: Error de sintaxis: Aridad de funciones invalido